common/src/enums/organizationApiKeyType.ts

```diff
@@ -0,0 +1,4 @@
+export enum OrganizationApiKeyType {
+  Default = 0,
```

Comment:
Yes, the `Default` type is for the api keys users already have on the `Organization` table. We will migrate them over to the database and they will get this type. `server` has the same `Default` option and it will be used to get them their "normal" api key for authenticating through the cli, api, etc.

Reply:
Confused this with `OrganizationConnectionType`, you're right.